### PR TITLE
FIX-FAILED-BESTPRACTICES: encrypt-running-vmss-linux

### PR DESCRIPTION
**Pull request by badri-spektra**
# PR Checklist

Check these items before submitting a PR... 

[Contribution Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md)

[Best Practice Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md)

- [ ] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

## Changelog

*
*
*

### CODE REVIEW

File: quickstarts/microsoft.compute/encrypt-running-vmss-linux/azuredeploy.parameters.json [hunk 1]
```diff
@@ -22,6 +22,9 @@
     },
     "forceUpdateTag": {
       "value": "1.0"
+    },
+    "valuturi": {
+        "value": "vault.azure.net/"
```
Comment (bmoore-msft) [inline quickstarts/microsoft.compute/encrypt-running-vmss-linux/azuredeploy.parameters.json:27]: remove this

File: quickstarts/microsoft.compute/encrypt-running-vmss-linux/azuredeploy.json [hunk 2]
```diff
@@ -48,22 +48,34 @@
...
-    }
+    },
+    "location": {
+      "type": "string",
+      "defaultValue": "[resourceGroup().location]",
+      "metadata": {
+        "description":"Default location"
+      }
+    },
+    "valuturi": {
```
Comment (bmoore-msft) [inline quickstarts/microsoft.compute/encrypt-running-vmss-linux/azuredeploy.json:59]: remove this and use the environment() function to get the uri

**Review by bmoore-msft — CHANGES_REQUESTED** [review]

File: quickstarts/microsoft.compute/encrypt-running-vmss-linux/azuredeploy.json (hunk 2 above)
Comment (badri-spektra) [reply quickstarts/microsoft.compute/encrypt-running-vmss-linux/azuredeploy.json]: updated with environment() function

File: quickstarts/microsoft.compute/encrypt-running-vmss-linux/azuredeploy.parameters.json (hunk 1 above)
Comment (badri-spektra) [reply quickstarts/microsoft.compute/encrypt-running-vmss-linux/azuredeploy.parameters.json]: removed

**Pull request merged**